api/src/main/java/org/openmrs/module/sync2/api/service/impl/SyncConfigurationServiceImpl.java

```diff
@@ -78,4 +100,27 @@ private String getConfigFilePath() {
         File configFileFolder = OpenmrsUtil.getDirectoryInApplicationDataDirectory(CONFIGURATION_DIR);
         return new File(configFileFolder, SYNC2_NAME_OF_CUSTOM_CONFIGURATION).getAbsolutePath();
     }
+
+    private SyncMethodConfiguration getSyncMethodConfiguration(SyncOperation operation) {
+        SyncMethodConfiguration configuration;
+        switch (operation) {
+            case PULL:
+                configuration = getSyncConfiguration().getPull();
+                break;
+            case PUSH:
+                configuration = getSyncConfiguration().getPush();
+                break;
+            default:
+                configuration = null;
+        }
+        return configuration;
+    }
+
+    private List<ClassConfiguration> getClassesConfiguration(SyncMethodConfiguration methodConfiguration) {
+        List<ClassConfiguration> result = new ArrayList<>();
+        if (methodConfiguration != null) {
+            result = methodConfiguration.getClasses();
+        }
+        return result;
```

Comment:
maybe use Optional instead a method with if statement

Reply:
I believe it won't improve complexity in this place.